api/src/adapters/db/clients/postgres_client.py

```diff
@@ -47,6 +47,7 @@ def get_conn() -> Any:
             "postgresql+psycopg://",
             pool=conn_pool,
             hide_parameters=db_config.hide_sql_parameter_logs,
+            execution_options={"schema_translate_map": db_config.get_schema_translate_map()}
```

Comment:
(non-blocking q) What's this for?

Reply:
Tables have a schema attached to them (in this case `api`) but what happens if you have different schemas depending on the environment? This map lets you tell SQLAlchemy to instead generate its queries with the alias. We only use this for tests, as we create schemas like `test_12345678_api` and instead do the tests in those to avoid conflicting with any data you might have locally.

Reply:
Note that the way we handled this before this change was by never specifying the schema, and instead setting the `search_path` when querying the DB, effectively changing what the queries could see. I removed the `search_path` logic as it actually breaks a lot when trying to use schemas this way (see latest commits)